lib/components/app/app-menu.js

```diff
@@ -170,21 +170,22 @@ class AppMenu extends Component {
           width='320px'
         >
           <ul className='app-menu'>
-            <MenuItem className='menu-item' onClick={this._showRouteViewer}>
-              <Icon name='bus' />
-              Route Viewer
+            {/* This item is duplicated by the view-switcher, but only shown on mobile
+            when the view switcher isn't shown (using css) */}
+            <MenuItem className='app-menu-route-viewer-link' onClick={this._showRouteViewer}>
+              <Icon type='bus' /> <FormattedMessage id='components.RouteView.shortTitle' />
```

Comment:
The message id should be `components.RouteViewer.shortTitle`.
Place `<FormattedMessage>` on the next line for consistency.

Reply:
Whoops! Thanks for noticing. Fixed in da3a686bd0dcfa48cfbdfb3b1052864eab9933fa